### ISSUE TITLE
[treyarc] [T6] missing animation syntax

### ISSUE THREAD

**Issue by xensik** [xensik/gsc-tool#221]
TODO: investigate namespace syntax for animtrees

sample:
```
open_hatch()
{
    self SetAnim( %vehicles::v_vtol_doors_open, 1, 0.1, 1 );
}
```

context:
![image](https://github.com/user-attachments/assets/bbf4f989-b12b-4228-ab33-a6ccd661ee3f)

**Comment by Jbleezy** [comment]
Are you saying it should have `vehicles::`?

Latest release (1.4.6) decompiles like this:

![image](https://github.com/user-attachments/assets/d79f81cd-a3b5-4cc7-90d6-6e0f282742b9)

**Comment by xensik** [comment]
yes @JezuzLizard found this missing syntax in SP scripts

**Comment by JezuzLizard** [comment]
Yeah SP makes use of this syntax because they want to reference multiple different animtrees in the same function. 
```
#using_animtree
```
is used as a file scope where functions defined after it implicitly resolve anims from that animtree. 

There is an alternative solution where you store the anims in variables since the animtree is only needed to be known when the anim is initialized. Although you'd still need to initialize the variables in separate functions as without the syntax that SP scripts use to explicitly reference an animtree.

**Comment by xensik** [comment]
fixed #227
